shared/actions/devices.js

```diff
@@ -16,11 +17,21 @@ export function loadDevices () : AsyncAction {
       param: {},
       incomingCallMap: {},
       callback: (error, devices) => {
-        dispatch({
-          type: Constants.showDevices,
-          payload: error || devices,
-          error: !!error
-        })
+        // Flow is weird here, we have to give it true or false directly instead of just giving it !!error
```

Comment:
Maybe update to .21?

On Wednesday, February 3, 2016, Marco Munizaga <notifications@github.com>
wrote:

> In shared/actions/devices.js
> <https://github.com/keybase/client/pull/1881#discussion_r51817029>:
>
> > @@ -16,11 +17,21 @@ export function loadDevices () : AsyncAction {
> >        param: {},
> >        incomingCallMap: {},
> >        callback: (error, devices) => {
> > -        dispatch({
> > -          type: Constants.showDevices,
> > -          payload: error || devices,
> > -          error: !!error
> > -        })
> > +        // Flow is weird here, we have to give it true or false directly instead of just giving it !!error
>
> yeah I'll make a ticket
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1881/files#r51817029>.
>